lib/littlebox.css

```diff
@@ -3261,26 +3261,26 @@
 .lb-flag-france {
```

Comment:
Aaah. Just found the misplace flag-canada note ha sorry about that.  I may never be allowed back into Paris. Thanks!